app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -23,8 +23,34 @@ public void onSignupClicked() {
         this.getView().goToSignupView();
     }
 
-    public void onLoginClicked() {
-        this.getView().goToHomePage();
+    public void onLoginClicked(String email, String password) {
+        if (!fieldsAreEmpty(email, password) && !emailIsInvalid(email)) {
```

Comment:
From these there are a lot of things we can talk about. We should talk about this tomorrow